maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java

```diff
@@ -60,6 +55,12 @@
 import org.eclipse.m2e.core.project.ResolverConfiguration;
 import org.eclipse.osgi.util.NLS;
 
+import com.liferay.ide.core.ILiferayConstants;
```

Comment:
Change import sorting to sort `com.liferay` above `java` and `org`.  Thanks

<img width="659" alt="screen shot 2017-10-26 at 1 36 39 am" src="https://user-images.githubusercontent.com/595221/32038126-26b9532c-b9ee-11e7-9341-3e8af29bccc7.png">